selectolax/node.pxi

```diff
@@ -115,11 +115,17 @@ cdef class _Attributes:
         return "<%s attributes, %s items>" % (tag_name, len(self))
 
 
+ctypedef fused str_or_Node:
+    str
+    Node
```

Comment:
Thank yo @rushter !
Yes, you were right. Adding explicit type fixed the issue.
I've used basestring instead of str/unicode for backward compatibility with Py2 and correct support of Py3.

According to the Cython docs:
```
The basestring type represents both the types str and unicode, i.e. all Python text string types in Python 2 and Python 3.
```

So we should be good. Tests are passing.